system/lib/libc/emscripten_exception_builtins.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+/*
+  References to these longjmp- and exceptions-supporting things are generated
+  in the llvm backend.
+
+  Note that these might make more sense in compiler-rt, but they need to be
+  built with multithreading support when relevant (to avoid races between
+  threads that throw at the same time etc.), and compiler-rt is not built
+  that way atm.
+*/
+
+int _Thread_local __THREW__ = 0;
+int _Thread_local __threwValue = 0;
```

Comment:
It looks like you can use the `thread_local` keyword if you include 

```
#include <threads.h>
thread_local int foo = 0;
```

I think that i slightly more readable and also the same keyword as C++ so less confusing.

Reply:
Makes sense, done.